scripts/site/_site/doc/app/app.routes.ts

```diff
@@ -1,16 +1,14 @@
 import { Routes } from '@angular/router';
-import { DEMOComponent } from './_demo/demo.component';
 
 import { DEMO_ROUTES } from './router';
 
 export const routes: Routes = [
   { path: '', pathMatch: 'full', redirectTo: '/docs/introduce/en' },
   ...DEMO_ROUTES,
   { path: 'docs', loadChildren: () => import('./docs/index.module').then(m => m.NzDocsModule) },
-  { path: 'demo', component: DEMOComponent },
```

Comment:
Will be redirected to `/introduce/zh` when visiting `/demo`, I wonder whether there are developer take advantage of this or not, so removed